src/overlays/actors/ovl_En_Pm/z_en_pm.c

```diff
@@ -833,9 +832,9 @@ void func_80AF8BA8(s32 arg0) {
         }
     }
 
-    temp = gSaveContext.save.weekEventReg[D_80AFB8E0[arg0] >> 8];
     gSaveContext.save.weekEventReg[D_80AFB8E0[arg0] >> 8] =
-        temp | (D_80AFB8E0[arg0] & (1 | 2 | 4 | 0x38 | 0x40 | 0x80));
+        ((void)0, gSaveContext.save.weekEventReg[D_80AFB8E0[arg0] >> 8]) |
+        (D_80AFB8E0[arg0] & (1 | 2 | 4 | 0x38 | 0x40 | 0x80));
```

Comment:
Isn't this just a `& 0xFF`?

Reply:
Huh, sure is. It isn't even there for logic, just there to help with regalloc. I just put a (u8) in front instead